backend/protos/xyz/block/ftl/v1/ftl.proto

```diff
@@ -184,9 +184,9 @@ message StreamDeploymentLogsRequest {
 message StreamDeploymentLogsResponse {}
 
 message StatusRequest {
-  bool all_runners = 1;
   bool all_controllers = 2;
-  bool all_ingress_routes = 3;
+
+  reserved 1, 3;
```

Comment:
Remove this, as Wes mentioned, we don't bother with preserving proto compatibility.

Reply:
got it, makes sense